pkg/component/component.go

```diff
@@ -842,13 +844,37 @@ func GetComponentDesc(client *occlient.Client, componentName string, application
 	if err != nil {
 		return componentDesc, errors.Wrap(err, "unable to get envVars list")
 	}
+
+	linkedServices := make([]string, 0)
```

Comment:
This creates a slice of size 0, if you want to create pre-allocate a size to avoid constant resizing when adding elements, you need to do `make([]string, 0, <expected max size>)`.

Reply:
Well the expected size is 0 to be honest.

Reply:
Fixed